daemon/src/maker_inc_connections.rs

```diff
@@ -136,10 +136,15 @@ impl Actor {
     async fn drop_taker_connection(&mut self, taker_id: &Identity) {
         if self.write_connections.remove(taker_id).is_some() {
             tracing::info!(%taker_id, "Dropping connection");
-            let _ = self
+            if self
                 .taker_disconnected_channel
                 .send(maker_cfd::TakerDisconnected { id: *taker_id })
-                .await;
+                .log_failure("Failed to inform about taker disconnect")
+                .await
+                .is_err()
+            {
+                tracing::warn!("TakerDisconnected receiver actor is disconnected");
```

Comment:
I am not sure we really need those? Bloats things quite a bit.

Reply:
I agree about the bloat, which is why my first thought was to include handling this in the `log_failure` trait. So you would just ignore these errors?

Reply:
At the moment we are ignoring them right? So I don't think we are degrading anything if we continue to ignore them!

It is hard to introduce changes like the current one if you are at the same time trying to change behaviour too much.

Reason I would ignore them in most cases is because they can never happen as by default, actors don't shut down in xtra. It is an explicit action.

Reply:
> It is hard to introduce changes like the current one if you are at the same time trying to change behaviour too much.

I agree on principal, but logging instead of silently dropping is never going to have unintended side-effects.

> Reason I would ignore them in most cases is because they can never happen as by default, actors don't shut down in xtra. It is an explicit action.

What if we explicitly shut down the receiving end and still want to send messages there? In this case I agree that we can ignore the error, but it seems wrong as a general rule.

Reply:
It the argument is that it can never happen, shouldn't we be `expect`ing?